.changeset/salty-tools-carry.md

```diff
@@ -1,9 +1,7 @@
 ---
 "fluid-framework": minor
 "@fluidframework/tree": minor
----
----
-"section": tree
+"__section": tree
```

Comment:
changesets have a new format, but the build-tools were only recently updated, so old changesets need to be manually converted.

Reply:
Ah good call. I think we should do this on its own PR though.

Reply:
Fair enough. One challenge is that new changesets are getting added with the old format - maybe I just need to make docs reviewers aware to watch for the old format and catch it in review.